src/ocm/components/clusterWizard/ClusterDetailsForm.tsx

```diff
@@ -147,6 +147,7 @@ const ClusterDetailsForm = (props: ClusterDetailsFormProps) => {
                 (dirty || (cluster && canNextClusterDetails({ cluster })))
               )
             }
+            isBackDisabled={isSubmitting}
```

Comment:
It is not needed since `ClusterDetails` is the first step and has no back button

Reply:
True! Nice catch